### PR TITLE
Add constant selector

### PR DESCRIPTION
**Pull request by piitaya**
### Usage

```yaml
white:
  name: White
  description: Set the light to white mode
  advanced: true
  selector:
    constant:
      value: true
      label: Enabled
```

### Form service

https://user-images.githubusercontent.com/5878303/224061490-358c6dc8-587a-4c54-8ed0-2358a764b226.mp4

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/data/selector.ts [hunk 1]
```diff
@@ -88,6 +89,13 @@ export interface ConfigEntrySelector {
   } | null;
 }
 
+export interface ConstantSelector {
+  constant: {
+    value?: string | number | boolean;
+    label?: string;
```
Comment (emontnemery) [inline src/data/selector.ts:95]: Should we support translating the label, like we do with the select selector options?

File: src/data/selector.ts [hunk 2]
```diff
@@ -88,6 +89,13 @@ export interface ConfigEntrySelector {
   } | null;
 }
 
+export interface ConstantSelector {
+  constant: {
+    value?: string | number | boolean;
```
Comment (emontnemery) [inline src/data/selector.ts:94]: Shouldn't this be mandatory?

File: src/data/selector.ts (hunk 1 above)
Comment (piitaya) [reply src/data/selector.ts:95]: I can be nice in a future PR. As services are not translated yet, we don't need it immediately.

Comment (piitaya) [reply src/data/selector.ts:95]: Yes.

What format do we want ? I propose this one.

```json
"selector": {
  "white": {
    "value": "Enabled"
  }
}
```

Comment (emontnemery) [reply src/data/selector.ts:95]: That seems reasonable 👍

**Review by bramkragten — APPROVED** [review]

**Pull request merged**